viewer/package.json

```diff
@@ -52,6 +52,7 @@
     "@types/tween.js": "^18.5.1",
     "@typescript-eslint/eslint-plugin": "^3.9.0",
     "@typescript-eslint/parser": "^3.9.0",
+    "core-js": "^3.6.5",
```

Comment:
could you share what problems did you have with the tests to add that?

Reply:
`Array.flat()` doesn't work in jest (node) without it

Reply:
Mmmmm.... which node version are you using?

```
Welcome to Node.js v12.18.3.
Type ".help" for more information.
> Array.prototype.flat
[Function: flat]
> [].flat
[Function: flat]
```

Should be fine after node > 11

Reply:
My jest is node 10 apparently (I use 12 for the rest of the toolchain though)

Reply:
Since it's just a dev dep I think it's ok to keep though